src/components/ArticleCollection.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState, useEffect } from "react";
+import Articles from "../modules/Articles";
+import useStyles from "../styles/styles";
+import {
+  Typography,
+  Container,
+  CardContent,
+  Grid,
+  Card,
+} from "@material-ui/core";
+
+const ArticleCollection = () => {
+  const [articles, setArticles] = useState([]);
+  const classes = useStyles();
+
+  useEffect(() => {
+    const fetchData = async () => {
```

Comment:
You don't define a function inside the "useEffect" hook. It doesn't make sense. You have to define it outside.
"fetchData" is generic, you can call it "fetchArticles".